x-pack/plugins/security_solution/public/management/components/artifact_entry_card/artifact_entry_card.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useMemo } from 'react';
+import { CommonProps, EuiHorizontalRule, EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
+import styled from 'styled-components';
+import { CardHeader, CardHeaderProps } from './components/card_header';
+import { CardSubHeader } from './components/card_sub_header';
+import { getEmptyValue } from '../../../common/components/empty_value';
+import { CriteriaConditions, CriteriaConditionsProps } from './components/criteria_conditions';
+import { EffectScopeProps } from './components/effect_scope';
+import { ContextMenuItemNavByRouterProps } from '../context_menu_with_router_support/context_menu_item_nav_by_rotuer';
+import { AnyArtifact } from './types';
+import { useNormalizedArtifact } from './hooks/use_normalized_artifact';
+import { useTestIdGenerator } from '../hooks/use_test_id_generator';
+
+const CardContainerPanel = styled(EuiPanel)`
+  &.artifactEntryCard + &.artifactEntryCard {
+    margin-top: ${({ theme }) => theme.eui.spacerSizes.l};
+  }
+`;
+
+export interface ArtifactEntryCardProps extends CommonProps {
+  item: AnyArtifact;
```

Comment:
Although this component accepts either an ExceptionListItem (also used by Event Filters) or Trusted app item, the entire component still needs some work to be able to truly support exceptions (ex. criteria conditions and per-policy defintions)